coronograph.txt

```diff
@@ -1,4 +1,5 @@
 - entrance pupil
+- WFS
```

Comment:
Actually, the WFS should maybe be not right where the coronagraph is.

Reply:
I see what you mean.